Baseline WARNINGS/ERROR in master

Since version 4.4.0 is available from the Adobe Maven Repo  baselining in latest master fails with one error and several warnings:

```
[ERROR] com.adobe.acs.commons.mcp.model: Version increase required; detected 4.2.0, suggested 4.3.0
[WARNING] com.adobe.acs.commons.cloudconfig: Excessive version increase; detected 4.4.1, suggested 4.4.0
[WARNING] com.adobe.acs.commons.cloudconfig: Version has been increased but analysis detected no changes; detected 4.4.1, suggested 4.4.0
[WARNING] com.adobe.acs.commons.filefetch: Excessive version increase; detected 4.4.1, suggested 4.4.0
[WARNING] com.adobe.acs.commons.filefetch: Version has been increased but analysis detected no changes; detected 4.4.1, suggested 4.4.0
[WARNING] com.adobe.acs.commons.marketo: Excessive version increase; detected 4.4.1, suggested 4.4.0
[WARNING] com.adobe.acs.commons.marketo: Version has been increased but analysis detected no changes; detected 4.4.1, suggested 4.4.0
[WARNING] com.adobe.acs.commons.marketo.client: Excessive version increase; detected 4.4.1, suggested 4.4.0
[WARNING] com.adobe.acs.commons.marketo.client: Version has been increased but analysis detected no changes; detected 4.4.1, suggested 4.4.0
[WARNING] com.adobe.acs.commons.mcp.form.workflow: Excessive version increase; detected 4.4.1, suggested 4.4.0
[WARNING] com.adobe.acs.commons.mcp.form.workflow: Version has been increased but analysis detected no changes; detected 4.4.1, suggested 4.4.0
[INFO] Baseline analysis complete, 1 error(s), 10 warning(s)
```
The error is a regression of https://github.com/Adobe-Consulting-Services/acs-aem-commons/commit/ee1674c994d4eb9b61e8596ededf82f87299bddd
The warnings are due to the fact that those packages are not versioned explicitly and therefore get the version of the bundle. Every exported package should get a dedicated package version!

--- a/bundle/src/main/java/com/adobe/acs/commons/mcp/model/ValueFormat.java
+++ b/bundle/src/main/java/com/adobe/acs/commons/mcp/model/ValueFormat.java
@@ -68,8 +68,8 @@ public enum ValueFormat {
         return humanReadableByteCount(bytes, false);
     }
     
- // From: https://programming.guide/worlds-most-copied-so-snippet.html
-    public static strictfp String humanReadableByteCount(long bytes, boolean si) {
+    // From: https://programming.guide/worlds-most-copied-so-snippet.html
+    private static strictfp String humanReadableByteCount(long bytes, boolean si) {
         int unit = si ? 1000 : 1024;
         long absBytes = bytes == Long.MIN_VALUE ? Long.MAX_VALUE : Math.abs(bytes);
         if (absBytes < unit) {

